snapcraft/internal/deprecations.py

```diff
@@ -25,7 +25,8 @@
 import logging
 
 _DEPRECATION_MESSAGES = {
-    'dn1': "The 'snap' keyword has been replaced by 'prime'."
+    'dn1': "The 'snap' keyword has been replaced by 'prime'.",
+    'dn2': "Custom plugins should now be placed in `snap/plugins`.",
```

Comment:
We sort of have a mix of markdown-style code blocks (the backticks) and single quotes, like above. Can we stick with either quotes or backticks in the deprecation notices? I have a slight preference to single quotes since these are surfaced directly to the user, but I have no strong feelings.